app/controllers/cinema_halls_controller.rb

```diff
@@ -1,69 +1,47 @@
-# frozen_string_literal: true
-
 class CinemaHallsController < ApplicationController
-  # GET /cinema_halls
   def index
-    cinema_halls = CinemaHall.all.map do |hall|
-      render_cinema_hall(hall)
-    end
-    render json: cinema_halls
+    cinema_halls = CinemaHalls::UseCases::FetchAll.new.call
```

Comment:
You don't need to create a use_case for that, just use your repository. Please use `CinemaHalls::Repository.new.find_all`.